pkg/storage/system_intake.go

```diff
@@ -41,9 +41,12 @@ func (s *Store) CreateSystemIntake(intake *models.SystemIntake) (*models.SystemI
 			process_status,
 			ea_support_request,
 			existing_contract,
+		    grt_review_email_body,
```

Comment:
I don't think we need these on creation of an intake. They'll be null if not assigned. Same is true for all the new fields.

Reply:
okay!